effects/BotanyInstanced.vvvvjs.fx

```diff
@@ -0,0 +1,131 @@
+#ifdef GL_ES
```

Comment:
honest question, as I don't know all the magic which is happening here: is this a shader that can be used for different applications, and should be shipped directly with VVVV.js, or is is more a domain specific shader for the terrain examples, and should rather be bundled with the examples?